schema-engine/core/src/state.rs

```diff
@@ -451,21 +451,6 @@ impl GenericApi for EngineState {
         .await
     }
 
-    async fn list_migration_directories(
-        &self,
-        input: ListMigrationDirectoriesInput,
-    ) -> CoreResult<ListMigrationDirectoriesOutput> {
-        let migrations_from_filesystem =
-            schema_connector::migrations_directory::list_migrations(Path::new(&input.migrations_directory_path))?;
-
-        let migrations = migrations_from_filesystem
-            .iter()
-            .map(|migration| migration.migration_name().to_string())
-            .collect();
-
-        Ok(ListMigrationDirectoriesOutput { migrations })
-    }
```

Comment:
We no longer need to issue a JSON-RPC command to list migration directories: we're doing it directly from TypeScript now.